docs/modules/plugins/pages/plugin-rest-api.adoc

```diff
@@ -17,6 +17,9 @@ implementation(group: 'org.vividus', name: 'vividus-plugin-rest-api', version: '
 
 == Properties
 
+:base-java-docs-url: https://docs.oracle.com/en/java/javase/11/
```

Comment:
does it make sense to put the baseline version of the java to the global attributes?

Reply:
@valfirst

Reply:
in general yes, but I'd like to do it in a separate PR, it will also include changes in "Date/Time plugin" docs